easybuild/tools/module_naming_scheme/hierarchical_mns.py

```diff
@@ -50,6 +50,8 @@
     'icc,ifort': ('intel', '%(icc)s'),
     'Clang,GCC': ('Clang-GCC', '%(Clang)s-%(GCC)s'),
     'CUDA,GCC': ('GCC-CUDA', '%(GCC)s-%(CUDA)s'),
+    'CUDA,icc,ifort': ('intel-CUDA', '%(icc)s-%(CUDA)s'),
+    'CUDA,intel': ('intel-CUDA', '%(intel)s-%(CUDA)s'),
```

Comment:
@akesandgren is this last entry also required? I would expect that only the `CUDA,icc,ifort` entry is required for the `iccifortcuda` toolchain?

You are not combining `intel` with `CUDA` anywhere, only `icc`+`ifort` with `CUDA`?